datafusion/sqllogictest/test_files/issue_17138.slt

```diff
@@ -0,0 +1,36 @@
+statement ok
```

Comment:
I recommend we name this file more specifically, or put it in an existing function, perhaps aggregate.slt ? 

Not a blocker, just a suggestion

Reply:
Yeah, maybe that would be better. But if there is already precedent for regression cases perhaps we can leave this PR as is and move it in a subsequent PR